src/snowcli/plugins/nativeapp/common_flags.py

```diff
@@ -11,7 +11,7 @@
 ForceOption = typer.Option(
     False,
     "--force",
-    help=f"""Defaults to unset. Passing in --force turns this to True, i.e. the CLI will implicitly respond “yes” to any prompts that come up.
-    This flag should be passed in if you are not in interactive mode and want to perform potentially destructive actions.""",
+    help=f"""Defaults to unset. If specified, allows the CLI to implicitly respond “yes” to any prompts that come up.
```

Comment:
When enabled, this option causes the command to implicitly approve any prompts that arise. 

You should enable this option if the `--interactive` option is not specified and if you want perform potentially destructive actions.

Reply:
A user can be in interactive mode in two ways: either we detect that they are in interactive mode in the code below, or they pass in the `--interactive` flag. This is why I did not make `--interactive` explicit in this description. Maybe we can rephrase to:
```
When enabled, this option causes the command to implicitly approve any prompts that arise.

You should enable this option if interactive mode is not specified and if you want perform potentially destructive actions.
```